components/VFBMain.js

```diff
@@ -1018,9 +1018,6 @@ export default class VFBMain extends React.Component {
     // Retrieve cookie for 'quick_help' modal
     var cookie = getCookie("show_quick_help");
     // Show 'Quick Help' modal if cookie to hide it is not set to True
-    if ((this.props.location.search.indexOf("id=") > -1) || (this.props.location.search.indexOf("i=") > -1) || (this.props.location.search.indexOf("q=") > -1)) {
```

Comment:
This should be restored since there was a card that stated that if the user used i= / id= / q= the quickhelp should not appear.